src/views/History.vue

```diff
@@ -1,9 +1,61 @@
 <template>
-  <div>
-    Studio zostało otworzone przez Józefa Grodzickiego w roku 1885. Pamiątką po początkowym okresie są zachowane zdjęcia przedstawiające 
-    ludzi wielu kręgów społecznych kierowanych chęcią uwiecznienia się. 
-  </div>
+  <v-container>
+    <v-layout  >
+      <v-flex lg6 md6 xs12 ma-5 >
+        <v-img
+                  :src="require(`@/assets/joseph-grodzicki.jpg`)"
```

Comment:
na mobilnych się rozjeżdża, trzeba by coś pokombinować żeby nie hardkodoawć width i height.

Jednym z atrybutów z jakim można by pokombinować to "contain" i max-height na przykład. Próbowałem się z tym wczoraj trochę pobawić,  wygląda to lepiej ale np znika mi obrazek całkowicie na mobilnych ;p